app/src/components/TodoList.jsx

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import { useState } from "react";
+
+import { theme } from "../theme";
+
+const {
+  colors: { shadow, black, blue },
+  borderRadius,
+} = theme;
+
+/**
+ * @type {React.FC<React.ComponentProps<"button">>}
+ */
+const Todo = ({ children, ...rest }) => {
+  const [focused, setFocused] = useState(false);
+  const [hovered, setHovered] = useState(false);
+
+  return ((
+    <button
+      onBlur={() => setFocused(false)}
+      onFocus={() => setFocused(true)}
+      onPointerOut={() => setHovered(false)}
+      onPointerEnter={() => setHovered(true)}
```

Comment:
To mozna chyba samym CSSem zrobic (`:hover`, `:focus`)?

Reply:
Good pick. I can then transition with between "✅" and "⬜" with CSS.

Reply:
https://github.com/ChopChopOrg/academy/pull/4#pullrequestreview-327363649